src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,51 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+import org.junit.jupiter.params.provider.ValueSource
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class CalculatorTest {
+
+    private val calculator = Calculator()
+
+    @DisplayName("계산식 성공")
+    @ParameterizedTest
+    @MethodSource("correctParameters")
+    fun `correctly calculate`(calculatorTestInput: CalculatorTestInput) {
```

Comment:
코틀린에서는 DisplayName을 사용하지 않고, 메서드 이름에 바로 한글을 사용할 수 있어요 :)
```suggestion
    @ParameterizedTest
    @MethodSource("correctParameters")
    fun `계산식 성공`(calculatorTestInput: CalculatorTestInput) {
```

Reply:
회사 컨밴션으로 쓰다보니 별 의심없이 사용해왔네요..
다음 테스트코드부터는 한글로 써볼게요!